natchez-ce3/src/main/scala/com/ovoenergy/natchez/extras/combine/ce3/IOLocalTrace.scala

```diff
@@ -28,4 +28,14 @@ class IOLocalTrace(private val local: IOLocal[Span[IO]]) extends Trace[IO] {
 
   override def traceUri: IO[Option[URI]] =
     local.get.flatMap(_.traceUri)
+
+  override def log(fields: (String, TraceValue)*): IO[Unit] = ???
```

Comment:
In `src/main` there shouldn't be any unimplemented methods.

Reply:
I'll work on implementing those today!